json/tbd_PSI_combo.json

```diff
@@ -18,14 +17,13 @@
       "rimjoist": 0.2,
       "parapet": 0.2,
       "fenestration": 0.2,
-      "concave": 0.2,
-      "convex": 0.2,
+      "corner": 0.2,
       "balcony": 0.3,
       "party": 0.2,
       "grade": 0.2
     }
   ],
-  "unit": [
+  "building": [
```

Comment:
Less confusing with regards to OpenStudio terminology e.g., a _housing unit_ vs _The building_.